### PR TITLE
Mannings2020

### PR DESCRIPTION
**Pull request by profxj**
Updates for the Mannings et al. 2020 paper.

### CODE REVIEW

**Review by SunilSimha — APPROVED** [review]
Looks good to me.

File: frb/data/FRBs/FRB121102.json [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
     "FRB": "FRB121102",
     "cosmo": "Planck15",
-    "dec": 33.14791666666667,
+    "dec": 33.14793155555555,
     "eellipse": {
-        "a": 0.1,
-        "b": 0.1,
-        "cl": 95.0,
-        "theta": 0.0
+        "a": 0.004,
```
Comment (keheintz) [inline frb/data/FRBs/FRB121102.json:14]: Need a_+b_sys now?

File: frb/galaxies/offsets.py [hunk 2]
```diff
@@ -25,14 +27,23 @@ def angular_offset(frb, galaxy, nsigma=5., nsamp=2000):
...
 
     # If systematic exists, add in quadrature
     if 'a_sys' in frb.eellipse.keys():
         sig_a = np.sqrt(frb.eellipse['a_sys']**2 + sig_a**2)
         sig_b = np.sqrt(frb.eellipse['b_sys']**2 + sig_b**2)
 
+    # Add in Galaxy error : THIS TREATMENT IS APPROXIMATE
+    if gal_sig is not None:
+        # Project
+        sig2_gal_a = gal_sig[1]**2 * np.cos(pa_ee)**2 + gal_sig[0]**2 * np.sin(pa_ee)**2
```
Comment (keheintz) [inline frb/galaxies/offsets.py:40]: sqrt around the gal + FRB uncertainty squares?

**Review by keheintz — APPROVED** [review]

File: frb/data/FRBs/FRB121102.json (hunk 1 above)
Comment (profxj) [reply frb/data/FRBs/FRB121102.json:14]: adding

File: frb/galaxies/offsets.py (hunk 2 above)
Comment (profxj) [reply frb/galaxies/offsets.py:40]: nope

**Pull request merged**